smarts/core/road_map.py

```diff
@@ -417,39 +481,58 @@ def parallel_roads(self) -> List[RoadMap.Road]:
 
         @property
         def lanes(self) -> List[RoadMap.Lane]:
+            """The lanes contained in this road."""
             raise NotImplementedError()
 
         def lane_at_index(self, index: int) -> RoadMap.Lane:
+            """Gets the lane with the given index."""
             raise NotImplementedError()
 
         def edges_at_point(self, point: Point) -> Tuple[Point, Point]:
+            """Get the boundry points perpendicular to the center of the road closest to the given
```

Comment:
I guess you need to do a search and replace on "boundry".  ;)

Reply:
Would you happen to be able to recommend a search tool that can also index the human brain? :brain: 

```suggestion
            """Get the boundary points perpendicular to the center of the road closest to the given
```